python/src/dynamic.rs

```diff
@@ -14,15 +15,13 @@ impl<G: GraphViewInternalOps + Send + Sync + 'static> DynamicGraphView for G {}
 #[derive(Clone)]
 pub struct DynamicGraph(Arc<dyn DynamicGraphView>);
 
-impl DynamicGraph {
-    pub fn new<G: GraphViewOps>(graph: G) -> DynamicGraph {
-        Self(Arc::new(graph))
-    }
+pub(crate) trait FromGraph {
+    fn into_dynamic(self) -> DynamicGraph;
 }
 
-impl From<Graph> for DynamicGraph {
-    fn from(value: Graph) -> Self {
-        Self(Arc::new(value))
+impl<G: GraphViewOps> FromGraph for G {
```

Comment:
This should be called IntoDynamic since it implements the into-style method?

Reply:
Done